src/VotePlace/VotePlaceManager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\VotePlace;
+
+use AppBundle\Entity\VotePlace;
+use AppBundle\Repository\VotePlaceRepository;
+
+class VotePlaceManager
+{
+    /** @var VotePlaceRepository VotePlaceRepository */
+    private $repository;
+
+    public function __construct(VotePlaceRepository $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function getVotePlaceWishesLabelsByIds(array $ids): array
+    {
+        return array_map(function (VotePlace $votePlace) {
```

Comment:
this method executes a query, hydrates a Place objects for keeping only name value
what about `queryBuilder->select(name)->getArrayResult()`